src/psyclone/tests/psyGen_test.py

```diff
@@ -2617,8 +2627,269 @@ def test_binaryoperation_can_be_printed():
     assert "BinaryOperation[operator:'+']\n" in str(binaryOp)
 
 
+# Test KernelSchedule Class
```

Comment:
Please add docstrings for the new tests.

Reply:
Done